tests/src/test/java/alluxio/IntegrationTestUtils.java

```diff
@@ -30,6 +31,34 @@
  * Util methods for writing integration tests.
  */
 public final class IntegrationTestUtils {
+  public static boolean isGcs(UnderFileSystem ufs) {
```

Comment:
I was thinking about this and I put it in `IntegrationTestUtils` because we are not using the `getUnderFSType` API in our codebase and this PR was to target integration tests. Since this PR is to address build failures, I tried to keep the scope as small as possible.